assets/css/custom.css

```diff
@@ -276,6 +275,72 @@ a:not(.button):hover {
 	}
 }
 
+/* DESKTOP/TABLET STICKY NAV BAR */
+/* Adds pseudo content before anchor ids so when you link to them, they aren't hidden by the sticky nav*/
+#speakers:target:before,
+#venue:target:before,
+#sponsors:target:before,
+#code-of-conduct:target:before,
+#faq:target:before {
+  content:' ';
+  display: block;
+  height: 40px;
+}
+
+.nav-section {
+	background-color: #172751;
```

Comment:
I think that this name is potentially confusing, because we have two different nav bars